civihr_default_theme/scss/pages/_admin_absence_tab.scss

```diff
@@ -1,47 +1,68 @@
-.chr_leave-absences-admin {
-  background: transparent;
-  box-shadow: none;
-  padding: 0;
-}
-
-.chr_absence-tab-page {
-
+.absence-tab-page {
   .button-container {
     margin-bottom: 20px;
   }
 
   .tabs-container {
+    .container {
+      /*to shift the container to left else it was not aligned with button*/
+      padding-left: 0;
+    }
 
     ul {
-      margin-left: 0px;
+      margin-left: 0;
+      /**/
       background: white;
 
       li {
-
         &:first-child {
+          /*to match the shift from left in the design*/
           margin-left: 20px;
         }
 
         a {
           font-size: 14px;
+          cursor: pointer;
+          /*otherwise it jumps in on hover*/
+          border-width: 0px;
+
+          &:hover {
+            /*as by default it sets it to #fbfbfc*/
+            background-color: transparent;
+            border-width: 0px;
+          }
+        }
+
+        &.active {
+          a {
+            &,
+            &:focus,
+            &:hover {
+              border: 0 !important;
```

Comment:
`!imporant` should be used only if necessary. Here I removed and it still works.